solanaj/src/main/java/org/p2p/solanaj/core/Account.kt

```diff
@@ -20,18 +16,29 @@ class Account {
         keyPair = TweetNaclFast.Signature.keyPair()
     }
 
-    constructor(secretKey: ByteArray) {
-        keyPair = TweetNaclFast.Signature.keyPair_fromSecretKey(secretKey)
+    constructor(keyPair: ByteArray) {
+        this.keyPair = TweetNaclFast.Signature.keyPair_fromSecretKey(keyPair)
     }
 
     private constructor(keyPair: TweetNaclFast.Signature.KeyPair) {
         this.keyPair = keyPair
     }
 
+    /**
+     * Returns an encoded [keyPair] in Base58
+     * */
+    fun getEncodedKeyPair(): String =
```

Comment:
you can return Base58String here and remove KDoc. code is more documented this way